metadata-ingestion/src/datahub/ingestion/source/openapi.py

```diff
@@ -61,6 +61,7 @@ class OpenApiConfig(ConfigModel):
     )
     forced_examples: dict = Field(default={}, description="")
     token: Optional[str] = Field(default=None, description="")
+    bearer_token: Optional[bool] = Field(default=False, description="Use Bearer token")
```

Comment:
so both token and bearer_token (1) can't be set at the same time and (2) use the same header

can we use a pydantic validator to check condition (1), and then set `token = f"Bearer {bearer_token}"`. that way, the rest of the code can stay oblivious to this difference